recorder/src/main/cpp/controller.cc

```diff
@@ -416,15 +429,15 @@ void populate_recording_header(recording::RecordingHeader& rh, const recording::
     *wa = w;
 }
 
-void Controller::issueWork(const std::string& host, const std::uint32_t port, std::uint32_t controller_id, std::uint32_t controller_version) {
+void Controller::issue_work(const std::string& host, const std::uint32_t port, std::uint32_t controller_id, std::uint32_t controller_version) {
     auto at = Time::now() + Time::sec(current_work.delay());
     scheduler.schedule(at, [&, port, controller_id, controller_version]() {
             with_current_work([&](Controller::W& w, Controller::WSt& wst, Controller::WRes& wres, Time::Pt& start_tm, Time::Pt& end_tm) {
                     auto work_id = w.work_id();
```

Comment:
Well, graphana (and not sure, perhaps opentsdb too) interpolates values. so the graph will be a mess (it'll start all sorts of confusion).

I don't think work-id as value is a good idea. We'll have logs for this purpose.